file/src/test/java/org/apache/zeppelin/file/HDFSFileInterpreterTest.java

```diff
@@ -34,6 +34,7 @@
 import org.apache.zeppelin.interpreter.InterpreterResult;
 import org.apache.zeppelin.interpreter.thrift.InterpreterCompletion;
 import org.junit.jupiter.api.Test;
+import org.slf4j.LoggerFactory;
```

Comment:
Please sort the imports here. I would delete `import org.slf4j.logger;` in line `26` and add it here.

Reply:
I have completed this task.